msticpy/config/query_editor.py

```diff
@@ -47,7 +47,8 @@ def _ipython_display_(self):
 
 
 def txt_fmt(width: str = "70%") -> Dict:
-    """Return a dictionary with layout and style for a text widget.
+    """
```

Comment:
Not a nit, but just curious: did we change the linter/formatter for all these docstring newlines?

Reply:
It's not a change. The format should have always been that - text on same line as quotes if only a single line, but on a new line if it's a multi-line docstring.
I unleashed Ruff's auto-fix, that did a lot of this.